src/test/scala/com/github/dnvriend/serializer/avro4s/AvroTestSpec.scala

```diff
@@ -126,4 +126,84 @@ class AvroTestSpec extends TestSpec {
   implicit class Base64Ops(base64: Base64) {
     def decode[OUT](implicit ev: Decoder[Base64, OUT]): OUT = ev.decode(base64)
   }
-}
\ No newline at end of file
+
+  object SchemaRegistry {
```

Comment:
The default avro serializer we use generates a default schema for the case class, I think that it doesn't use a schema version. Maybe we should investigate this.